project/Build.scala

```diff
@@ -1349,6 +1349,17 @@ object Build {
       testHtmlSettings(testHtmlFullOpt, FullOptStage) ++ Seq(
         name := "Scala.js test suite",
 
+        unmanagedSourceDirectories in Test ++= {
+          def includeIf(testDir: File, condition: Boolean): List[File] =
+            if (condition) List(testDir)
+            else Nil
```

Comment:
Do you repeat this on purpose? Maybe lift it so you don't have to define it twice?